microsoft-edge/webview2/how-to/javascript.md

```diff
@@ -67,11 +67,14 @@ For example:
 ## Scenario: Running a dedicated script file
 
 In this section, you access a dedicated JavaScript file from your WebView2 control.
+<!-- todo: clarify the scenario.  from native code? "from" is ambiguous.  is the file a web-side file?
```

Comment:
clarify the scenario. from native code? "from" is ambiguous. is the file a web-side file?